lib/stripe/plans.rb

```diff
@@ -86,8 +89,11 @@ def create_options_without_products
           interval_count: interval_count,
           trial_period_days: trial_period_days,
           metadata: metadata,
+          usage_type: usage_type,
```

Comment:
I'm removing these lines since these are the deprecated options for the old API and do not need to be updated. I'll make a note on the method.

Unfortunately the [Stripe API reference](https://stripe.com/docs/api) only shows me the latest version of the API so I cannot confirm that these are actually invalid.

Reply:
That makes sense.